pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -0,0 +1,210 @@
+package mapper
+
+import (
+	"strconv"
+
+	v2vv1alpha1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1alpha1"
+	provider "github.com/kubevirt/vm-import-operator/pkg/providers"
+	"github.com/kubevirt/vm-import-operator/pkg/utils"
+	ovirtsdk "github.com/ovirt/go-ovirt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+)
+
+const (
+	cdiAPIVersion = "cdi.kubevirt.io/v1alpha1"
+)
+
+var biosMapping = map[string]*kubevirtv1.Bootloader{
```

Comment:
It would be good to use this mapping in validation - to have one source of truth.